src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -119,4 +119,12 @@ public static String getHostForWiki() {
   public static String getUrlForPage(String pageName) {
     return getHostForWiki() + pageName;
   }
+
+  public static String getUrl(String wikiName, String path) {
+    String environment = Configuration.getEnv();
+
+    environment = environment.equals("prod") ? "" : environment + ".";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "prod" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1132)